### DISCUSSION TITLE
Iterating a dataframe and adding one column as a key and the other as value

### DISCUSSION THREAD

**Discussion by Oceanestars** [opensource9ja/danfojs#427] [Q&A]
Hi!
I'm struggling to migrate from pandas to danfo.
I'm trying to just get the left column to be key of a dictionary and right column to be values.
```javascript
 var numOfAppearance = df["Title"].valueCounts().sortValues({ "ascending": false});
 numOfAppearance.print(); //this works great and has more than 5 rows hence why i'm using iloc after
 
var temp = numOfAppearance.iloc([0,1,2,3,4]).print(); //temp is undefined... I'm not sure why but the right side of the equal sign does select ny first 5 rows. 
console.log(temp);
```

Here I'm getting the correct data frame with the right data but when I'm trying to just select the first-row title It's not working. So I tried to go back even a step before where I could just get the column and all the titles of that column. Ideally, I can iterate through the title column of the 5 rows and make each value be the key to a dictionary but I'm not able to iterate through the data frame. Any help is greatly appreciated!!

Thank you!

**Comment by risenW** [comment]
@Oceanestars Is your index numeric or string? And also if you need just the first row, you can do `var temp = numOfAppearance.iloc([0])`
